### PR TITLE
Add PodcastIndex helipad

### PR DESCRIPTION
**Pull request by Kukks**

### CODE REVIEW

File: Production/nginx.tmpl [hunk 1]
```diff
@@ -107,6 +107,19 @@
         sub_filter 'href="../'  'href="/bluewallet_lndhub_app/';
         sub_filter 'src="/'  'src="/bluewallet_lndhub_app/';
         sub_filter_once off;
+	}
+		{{ end }}
+		{{ if (eq $serviceName "helipad") }}
+	location /helipad/ {
+		proxy_set_header Connection "";
+		proxy_set_header        Host               $host;
+        proxy_set_header        X-Real-IP          $remote_addr;
```
Comment (dergigi) [inline Production/nginx.tmpl:116]: Inconsistent indentation?

File: docker-compose-generator/docker-fragments/opt-add-helipad.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+  btcpayserver:
+    environment:
+      BTCPAY_EXTERNALSERVICES: "Podcast Index Helipad:${BTCPAY_PROTOCOL:-https}://${BTCPAY_HOST}/helipad;"
+
+volumes:
+  helipad_datadir:
+        
+required:
+- "bitcoin-lnd"
```
Comment (dergigi) [inline docker-compose-generator/docker-fragments/opt-add-helipad.yml:25]: No newline at EOF?

**Review by dergigi — COMMENTED** [review]
Looks good to me at a first glance.

NIT: two minor formatting inconsistencies.

File: Production/nginx.tmpl (hunk 1 above)
Comment (Kukks) [inline Production/nginx.tmpl:116]:
```suggestion
		proxy_set_header        X-Real-IP          $remote_addr;
```

File: docker-compose-generator/docker-fragments/opt-add-helipad.yml (hunk 2 above)
Comment (Kukks) [inline docker-compose-generator/docker-fragments/opt-add-helipad.yml:25]:
```suggestion
- "bitcoin-lnd"

```

**Comment by NicolasDorier** [comment]
Document it in the README, then seems good.

**Comment by Kukks** [comment]
> Document it in the README, then seems good.

already there

**Pull request merged**